bindings/java/src/test/java/org/openucx/jucx/UcpEndpointTest.java

```diff
@@ -241,6 +242,51 @@ public void onSuccess(UcxRequest request) {
         context2.close();
     }
 
+    @Test
+    public void testRecvAfterSend() {
+        // Crerate 2 contexts + 2 workers
+        UcpParams params = new UcpParams().requestRmaFeature().requestTagFeature();
+        UcpWorkerParams rdmaWorkerParams = new UcpWorkerParams().requestWakeupRMA();
+        UcpContext context1 = new UcpContext(params);
+        UcpContext context2 = new UcpContext(params);
+        UcpWorker worker1 = context1.newWorker(rdmaWorkerParams);
+        UcpWorker worker2 = context2.newWorker(rdmaWorkerParams);
+
+        UcpEndpoint ep = worker1.newEndpoint(new UcpEndpointParams()
+            .setUcpAddress(worker2.getAddress()));
+
+        ByteBuffer src1 = ByteBuffer.allocateDirect(UcpMemoryTest.MEM_SIZE);
+        ByteBuffer dst1 = ByteBuffer.allocateDirect(UcpMemoryTest.MEM_SIZE);
+
+        ep.sendTaggedNonBlocking(src1, 0, null);
+
+        try {
+            Thread.sleep(5);
```

Comment:
need to mix sleep with several calls to progress like in gtest
potentially 1 progress call will not be enough to complete the message
also, maybe progress form another thread to test thread safety?